### PR TITLE
Dependency cron 2020-10-09

### PR DESCRIPTION
**Pull request by openqabot**

### CODE REVIEW

File: t/40-job_settings.t [hunk 1]
```diff
@@ -47,7 +47,7 @@ my $settings = {
     HDDSIZEGB                => 50,
     INSTALLONLY              => 1,
     PATTERNS                 => 'base,minimal',
-    PUBLISH_PFLASH_VARS =>
+    PUBLISH_PFLASH_VARS      =>
       'SLES-%VERSION%-%ARCH%-%BUILD%@%MACHINE%-minimal_with_sdk%BUILD_SDK%_installed-uefi-vars.qcow2',
     SEPARATE_HOME      => 0,
```
Comment (coolo) [inline t/40-job_settings.t:52]: And on next release it will align this arrow too?

Comment (okurz) [reply t/40-job_settings.t:52]: one at a time, don't rush it :D

I guess this shows all the critics that Perl is far from a dead language, right? ;)

**Review by okurz — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Review by okurz — CHANGES_REQUESTED** [review]
hm, OBS checks fail now but in other pull requests they are fine. I think someone needs to investigate further.

**Review by okurz — APPROVED** [review]
hmpf, other PRs fail like https://app.circleci.com/pipelines/github/os-autoinst/openQA/4472/workflows/0006b416-be39-4b35-8a72-93508eebca51/jobs/42847 because the old tidy package already vanished. This setup sucks … So we have no other chance than accept this PR now.

**Pull request merged**